client/src/connection/rest/index.ts

```diff
@@ -54,10 +59,26 @@ async function setup() {
 
   if (config.serverId) {
     const server1 = new ComputeServer(config.serverId);
+
+    //if we have a session already, get it
+    let sessionId: string = await getContextValue("SAS.sessionId");
```

Comment:
I think there's still more to discuss here around session IDs and a session interface. Ultimately, I'd love to see a function on a `Session` class/interface/etc because it makes sense conceptually (i.e. "a session has an identity"). 

For now, I think this is a good addition as we'll always likely need context to persist our session id across reloads/etc. With this in place, I _could_ rely on `getContextValue('SAS.sessionId')` to give me the session id I need (assuming that could also change as sessions evolve)

Reply:
Yeah, you could definitely use this to get the sessionId and it could easily change as the session interface changes.